hugo/content/playground/textmate-generator.ts

```diff
@@ -0,0 +1,255 @@
+/******************************************************************************
+ * Copyright 2021-2022 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+******************************************************************************/
+//ATTENTION!!! This file is basically copied from langium-cli!
```

Comment:
Should we export that in `langium-cli`? It is currently not available because it is not part of any export, correct?

Reply:
[On its way](https://github.com/eclipse-langium/langium/pull/1434)...